sentry/src/main/java/io/sentry/logger/LoggerApi.java

```diff
@@ -231,6 +236,24 @@ private void setServerName(
     }
   }
 
+  private void setUser(final @NotNull HashMap<String, SentryLogEventAttributeValue> attributes) {
+    final @Nullable User user = scopes.getCombinedScopeView().getUser();
+    if (user != null) {
+      final @Nullable String id = user.getId();
+      if (id != null) {
+        attributes.put("user.id", new SentryLogEventAttributeValue("string", id));
```

Comment:
```suggestion
        attributes.put("user.id", new SentryLogEventAttributeValue(SentryAttributeType.STRING, id));
```
I think we'll want to deprecate the String constructor in the future, might as well use this one directly.